src/app/forms/templates/test-records/section-templates/test/test-section-group1.template.ts

```diff
@@ -46,13 +46,103 @@ export const TestSectionGroup1: FormNode = {
             {
               name: 'testResult',
               label: 'Result',
-              value: '',
+              editType: FormNodeEditTypes.RADIO,
+              options: [
+                { value: 'fail', label: 'Fail' },
+                { value: 'pass', label: 'Pass' },
+                { value: 'prs', label: 'PRS' },
+                { value: 'abandoned', label: 'Abandoned' }
+              ],
               validators: [
-                { name: ValidatorNames.enableIfEquals, args: { sibling: 'reasonForAbandoning', value: 'abandoned' } },
-                { name: ValidatorNames.enableIfEquals, args: { sibling: 'additionalCommentsForAbandon', value: 'abandoned' } }
+                { name: ValidatorNames.HideIfNotEqual, args: { sibling: 'reasonForAbandoning', value: 'abandoned' } },
+                { name: ValidatorNames.HideIfNotEqual, args: { sibling: 'additionalCommentsForAbandon', value: 'abandoned' } }
               ],
               type: FormNodeTypes.CONTROL
             },
+            {
+              name: 'reasonForAbandoning',
+              type: FormNodeTypes.CONTROL,
+              label: 'Reason for abandoning',
+              editType: FormNodeEditTypes.CHECKBOX,
+              separator: '. ',
+              required: true,
+              validators: [
+                {
+                  name: ValidatorNames.RequiredIfEquals,
+                  args: { sibling: 'testResult', value: 'abandoned' }
+                }
+              ],
+              options: [
```

Comment:
Might be worth extracting this list out into some global constant file to keep this template easier to read.